### ISSUE TITLE
Any documentation on this Unreal Engine FB integration?

### ISSUE THREAD

**Issue by yhyu13** [xiaodaiacg/fastbuild-UE#1]
Hi, I'm working on using FB for UE4 in free time, but it seems the original release of FBv1.07 makes build time 3 times slower than UBT's default Parallel Executor, and it consumes a hell lot more memories. I speculate that FB fails to understand Unreal's PCH correctly:  https://github.com/fastbuild/fastbuild/issues/823. 

If I understand your commits correctly, you have integrated the UnrealEngine\Engine\Extras\ThirdPartyNotUE\FASTBuild\fbuild_0.99.diff into FB v1.07?

**Comment by xiaodaiacg** [comment]
> If I understand your commits correctly, you have integrated the UnrealEngine\Engine\Extras\ThirdPartyNotUE\FASTBuild\fbuild_0.99.diff into FB v1.07?

yes, I just integrated some code from fbuild_0.99.diff, and it works fine for building UE5. There is noting more changed from FB original code. So I think documentation is not necessary, sorry.
